### ISSUE TITLE
HmIP-ESI nicht verfürbar

### ISSUE THREAD

**Issue by reihd** [ioBroker/ioBroker.hm-rpc#1087]
Hallo,

das neue Gerät HmIP-ESI ist leider nicht verfügbar. Dieses ist für das auslesen der Stromzähler. Vielleicht kann das in die nächste Version mit aufgenommen werden.

**Comment by reihd** [comment]
ich habe mit den neuen Energiezähler gekauft und dieser läuft auch auf der CCU3, aber ich sehe diesen nicht im iobroker

**Comment by Apollon77** [comment]
Und du hast nachdem hinzufügen einmal in der hm.-rpc Konfiguration  "Geräte neu synchonisieren" gewählt?

**Comment by reihd** [comment]
ja, hatte ich gemacht. ABER: ich habe es gefunden, die Namen sind halt falsch, ich habe auch einen Homatic Gaszähler und der Stromzähler wird auch als Gas angezeigt, auch die Formate passen nicht so richtig, steht alles in Wh und nicht in kwh, aber damit kann man leben, auf alle Fälle hast du mich weiter gebracht und ich kann die Werte schon mal sehen, damit kann das Ticket wohl zu und vielleicht gibt es beim Update irgendwann die korrekten Namen

**Comment by foxriver76** [comment]
Ja vermutlich macht die CCU Oberfläche ne Konvertierung von wh nach kwh aber über api nicht, finde ich jetzt auch nicht ganz unsinnvoll.

und ja Namen holt rega, im Zweifel mal neustarten.

**Comment by reihd** [comment]
neu gestartet habe ich alles schon. Aber bleibt halt beim Gas, die Namen sind auch egal, es ist auch sehr schade, dass HM die Daten nicht alle über die API übergibt, dann hätte man Tageswerte, Wochenwerte usw. aber so muss man sich die halt selbst erstellen. Ich bin ja schon zufrieden, dass ich die wesentlichen Werte bekomme

**Comment by GermanBluefox** [comment]
Die Tages-, Wochen- und Monatswerte berechnet die CCU selbst und legt sie als interne Systemvariablen an, über die RPC-Schnittstelle des Geräts gibt es sie nicht. In ioBroker kommen sie über hm-rega, wenn dort auch die internen Systemvariablen übertragen werden. Die Namen der Geräte setzt ebenfalls hm-rega.
